src/common/imageio_rawspeed.cc

```diff
@@ -120,7 +120,7 @@ dt_imageio_retval_t dt_imageio_open_rawspeed(dt_image_t *img, const char *filena
 
 #ifdef __WIN32__
   const size_t len = strlen(filename) + 1;
-  wchar_t filen[len];
+  wchar_t filen[MAX_PATH];
   mbstowcs(filen, filename, len);
```

Comment:
`mbstowcs()` needs len only to know how big output array (`filen`) is. 
So if you just properly pass either `MAX_PATH` or equivalent `sizeof(filen) / sizeof(wchar_t)` (latter is preferred.), it should be just fine, no?